<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGameObject</class>
 <widget class="QDockWidget" name="QGameObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGameObject</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_ObjectEnable">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_ObjectPhysics">
       <property name="text">
        <string>Physics</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_ObjectPhysics">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_ObjectID">
       <property name="text">
        <string>Obect ID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_ObjectID">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_ObjectName">
       <property name="text">
        <string>Object Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_ObjectName"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_ObjectParent">
       <property name="text">
        <string>Parent</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBox_ObjectParent"/>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_ObjectEnable">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
